### ISSUE TITLE
[BUG] Remote non-github Git integration gives me Github related errors when trying to use VC from mage

### ISSUE THREAD

**Issue by Mithrandir2k18** [mage-ai/mage-ai#4757]
### Mage version

Latest docker image: e5a0f90f6e1a

### Describe the bug

I have set up a monorepo for our projects and intend to use local mage instances for development and use our azure devops instance as host for our remote git repo. Mage did pickup the remote correctly. SSH-keys are ed25519 and do not have a password on them. When I add the base64 encoded keys to the relevant fields, all seems to be working fine, but when I try to e.g. 

Version Control --> Remote & Auth --> Fetch, origin

I get the following error:

Please authenticate with Github before trying to fetch.

### To reproduce

1. Clone a non-github git repo with SSH
2. cd into the repo
3. run `selected_dir=testproject && mkdir -p $selected_dir && docker run -it -p 6789:6789 -v $(pwd):/home/src mageai/mageai /app/run_app.sh mage start $selected_dir`
4. Add your ssh-keys and try to fetch from repo

### Expected behavior

Fetch from repo

### Operating system

Ubuntu 20.04, but I use docker and the latest mage image.

**Comment by dy46** [comment]
hey @Mithrandir2k18, the Version Control app will only work with OAuth authentication. Are you able to authenticate with your remote repo through OAuth instead of SSH?

**Comment by Mithrandir2k18** [comment]
> hey @Mithrandir2k18, the Version Control app will only work with OAuth authentication. Are you able to authenticate with your remote repo through OAuth instead of SSH?

No, sadly our organization only allows git via SSH. My workaround is currently to manage git manually outside the container, which led me to discover issue #4804 , but can be mitigated by chowning before any git operations or using git with sudo.
